src/open_inwoner/accounts/forms.py

```diff
@@ -279,6 +279,17 @@ def send_mail(
         email_message.send()
 
 
+class CategoriesForm(forms.ModelForm):
+    selected_categories = forms.ModelMultipleChoiceField(
+        queryset=Category.objects.published(),
```

Comment:
@alextreme willen we subonderwerpen tonen in het formulier? Of alleen root nodes?

Reply:
@stevenbal volgens mij waren alle categories beschikbaar, dus incl subonderwerpen